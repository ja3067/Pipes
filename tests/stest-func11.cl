def foo(x : list) -> float:
    def bar(x : list) -> int:
	return 3

    type(bar([4, 2])) # int
    
    def bar():
	return 3.0
    
    type(bar()) # float
    
    def bar(x) -> float:
	return 3.0
    
    type(bar("hello")) # float

    type(bar) # func

    bar("hello")
    return 3.0

type(foo([1, 2, 3])) # float

