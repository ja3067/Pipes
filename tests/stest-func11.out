int
float
float
func
float
int
float
float
func
